src/bin/wc.rs

```diff
@@ -104,23 +104,48 @@ struct Flags {
 }
 
 impl Flags {
-    fn print_count<'a, W: Write>(self, count: Counter, file: &'a str, stdout: &mut W, stderr: &mut Stderr) {
-        stdout.write(b"    ").try(&mut *stderr);
+    fn print_counts<W: Write>(self, mut counts: Vec<(Counter, String)>, stdout: &mut W, stderr: &mut Stderr) {
```

Comment:
No that's not _at all_ what I mean. What is say: is that you do another unnecessary allocation. 2 extra usizes is not a problem. Alloction is.